### ISSUE TITLE
[Docusaurus] Code freeze for the patterns project

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/green-software-patterns#163]
Hi @osamajandali a question came up in today's team call about Docusaurus. We will announce code freeze in the patterns repo at the end of the day today (PT time). @dubrie explained to me that Docusaurus is currently reading from the dev branch. He was going to merge dev to main and then announce the code freeze. So he was wondering if and how this would affect docusaurus. 

@osamajandali can you help with this?
@dubrie correct me if I misunderstood or missed anything

**Comment by osamajandali** [comment]
We only need to change the deploy github action to be reading from the main branch not the dev branch. I can do it once the merge is done.

**Comment by holanita** [comment]
Great thanks! @dubrie let us know when the merge is done and you are ready.

**Comment by dubrie** [comment]
Dev has been merged into main, you can change the github action now.  Thanks @osamajandali and @holanita!

**Comment by holanita** [comment]
@zanete @osamajandali please discuss in today's call and if this is just something minor, pls complete asap. thanks!

**Comment by osamajandali** [comment]
This one is done

**Comment by holanita** [comment]
Wonderful thanks a lot @osamajandali let me double check something with @dubrie by tomorrow before we close this

**Comment by holanita** [comment]
As discussed in today's meeting this is all fine now. From now on we should commit to dev. 
Bill asked for a preview of the dev branch, something like dev.greensoftware... (Asim will figure this out)
To be continued here: https://github.com/Green-Software-Foundation/green-software-patterns/issues/174
